engarde/checks.py

```diff
@@ -246,7 +247,30 @@ def one_to_many(df, unitcol, manycol):
     return df
 
 
-__all__ = ['is_monotonic', 'is_shape', 'none_missing', 'unique_index', 'within_n_std',
-           'within_range', 'within_set', 'has_dtypes',
-           'verify', 'verify_all', 'verify_any']
+def is_same_as(df, df_to_compare, **kwargs):
+    """
+    Assert that two pandas dataframes are the equal
+
+    Parameters
+    ==========
+    df : pandas DataFrame
+    df_to_compare : pandas DataFrame
+    **kwargs : dict
+        keyword arguments passed through to panda's ``assert_frame_equal``
+
+    Returns
+    =======
+    df : DataFrame
+
+    """
+    try:
+        tm.assert_frame_equal(df, df_to_compare)
+    except AssertionError as e:
+        raise AssertionError("DataFrames are not equal")
```

Comment:
It'd be nice to include the original assertion error for why they aren't equal here. I know in python3 it's just

```python
raise AseertionError("DataFrames are not equal") from e
```

but I don't know about py2. I think leave it for now.

Are you using python3 or 2? I'm tempted to just make the next release python3 only.